typescript/src/services/TsParser/CustomCompilerHost.ts

```diff
@@ -23,7 +23,7 @@ export class CustomCompilerHost implements CompilerHost {
       return createSourceFile(path.relative(this.baseDir, resolvedPath), text, languageVersion);
     } catch (e) {
       // if it is a folder then try loading the index file of that folder
-      if (e.code === 'EISDIR') {
+      if ((e as NodeJS.ErrnoException)?.code === 'EISDIR') {
```

Comment:
Why the `?` here? Surely once you have cast it, it is defined?

Reply:
Good point :+1: